sdk/src/AbstractMain.cpp

```diff
@@ -238,7 +275,26 @@ int AbstractMain::run(int in_argc, char** in_argv)
                getProgramId(),
                options.getScratchPath())));
    }
+   // Ensure logging directory path exists and is configured correctly.
+   int ret_log_dir = configureLoggingDir(options.getLoggingDir(), serverUser, options.useUnprivilegedMode());
+   if (ret_log_dir != 0)
+      return ret_log_dir;
 
+   if (options.getEnableDebugLogging())
+   {
+      options.getLoggingDir().changeFileMode("ALL_READ_WRITE_EXECUTE");
```

Comment:
I think we can completely delete the previous `FileLogDestination` above, since we are creating the debug logging file log destination here now. Thoughts @MariaSemple?

Reply:
The logging destination above this one is for the scratch path. Is it no longer necessary ?

Reply:
Yes. The logging dir defaults to scratch path. The other log destination I am referring to seemed to just be an additional file that gets logged if the log level is set to debug. We can get rid of it because our logging has evolved a bit since then and we specifically have `enable-debug-logging` and `logging-dir` which you are looking at to set things up here.